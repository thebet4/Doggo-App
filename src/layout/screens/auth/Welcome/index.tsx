import React from "react";
import { Content, TitleText, WaveContainer } from "./styles";
import { Loading } from "@layout/components/atoms";

const WelcomeScreen = () => {
  return (
    <WaveContainer source={require("../../../../../assets/images/splash.png")}>
      <Content>
        <TitleText>Welcome to Doggo</TitleText>
        <Loading size="large" />
      </Content>
    </WaveContainer>
  );
};

export default WelcomeScreen;
