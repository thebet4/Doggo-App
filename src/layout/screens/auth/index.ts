export { default as WelcomeScreen } from "./Welcome";
export { default as LoginScreen } from "./Login";
