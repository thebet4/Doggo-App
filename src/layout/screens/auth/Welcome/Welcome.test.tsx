import React from "react";
import renderer from "react-test-renderer";

import { WelcomeScreen } from "@layout/screens/auth";

it(`renders <Container /> correctly`, () => {
  const tree = renderer.create(<WelcomeScreen />).toJSON();

  expect(tree).toMatchSnapshot();
});
