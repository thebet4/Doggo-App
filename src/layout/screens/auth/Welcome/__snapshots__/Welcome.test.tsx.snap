// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders <WelcomeScreen /> correctly 1`] = `
<View
  accessibilityIgnoresInvertColors={true}
  style={
    Array [
      Object {
        "height": "100%",
        "width": "100%",
      },
    ]
  }
>
  <Image
    source={1}
    style={
      Array [
        Object {
          "bottom": 0,
          "left": 0,
          "position": "absolute",
          "right": 0,
          "top": 0,
        },
        Object {
          "height": "100%",
          "width": "100%",
        },
        undefined,
      ]
    }
  />
  <RNCSafeAreaView
    edges={
      Array [
        "bottom",
        "left",
        "right",
        "top",
      ]
    }
    style={
      Array [
        Object {
          "alignItems": "center",
          "flexBasis": 0,
          "flexGrow": 1,
          "flexShrink": 1,
          "justifyContent": "flex-end",
          "marginBottom": 64,
        },
      ]
    }
  >
    <Text
      style={
        Array [
          Object {
            "color": "#FFFFFF",
            "fontFamily": "Inter_600SemiBold",
            "fontSize": 16,
            "marginBottom": 64,
          },
        ]
      }
    >
      Welcome to Doggo
    </Text>
    <ActivityIndicator
      color="#011B30"
      size="large"
      style={
        Array [
          Object {},
        ]
      }
    />
  </RNCSafeAreaView>
</View>
`;
